docs: Update 'user guides' -> 'client apps' (#5940)
CloudFire-LLC/cloudfire: website/src/app/kb/deploy/users/readme.mdx

</Alert>

Users with the `User` role can only sign in to the Firezone Clients to access
Resources, while users with the `Admin` role can sign in to the admin portal to
perform administrative tasks as well as sign in to Firezone Clients to access
Resources.

> After creating a user, you can choose to assign the user to one or more groups.

Reviewer: ```suggestion
After creating a user, you can assign the user to one or more groups.
```

> This will ultimately determine the policies that apply to the user, so it's

Reviewer: ```suggestion
Groups determine the policies that apply to the user, so it's
```
